pkg/cmd/roachtest/tests/schemachange_random_load.go

```diff
@@ -151,12 +151,15 @@ func saveArtifacts(ctx context.Context, t test.Test, c cluster.Cluster, storeDir
 	defer db.Close()
 
 	// Save a backup file called schemachange to the store directory.
-	_, err := db.Exec("BACKUP DATABASE schemachange to 'nodelocal://1/schemachange'")
+	_, err := db.Exec("BACKUP DATABASE schemachange INTO 'nodelocal://1/schemachange'")
 	if err != nil {
 		t.L().Printf("Failed execute backup command on node 1: %v\n", err.Error())
 	}
-
-	remoteBackupFilePath := filepath.Join(storeDirectory, "extern", "schemachange")
+	var backupPath string
+	if err := db.QueryRow("SELECT path FROM [SHOW BACKUPS IN 'nodelocal://1/schemachange']").Scan(&backupPath); err != nil {
```

Comment:
you may want to run this manually to make sure this worked

Reply:
Verified, works!